michaeljevans/Crate
# Feature/sn survey content

#### What's this PR do?
- Creates the mens and womens style surveys and imports them in the Survey component
- Adds images and survey content: questions, values, image description
- Adds styling for image size
#### Where should the reviewer start?
Review the `men-survey.js` and `women-survey.js` files in `module/common/surveys' to see the survey content
#### How should this be manually tested?
In the browser, click on a `Subscribe` button on the Crates page. You should see the mens survey displayed and be able to complete the survey. 
#### Any background context you want to provide?
- Only the mens survey is currently showing as it's hardcoded.
- There's room to improve the image sizes but since @GreysonElkins is also working on some styling for the survey I figure it'll be more efficient to come back to this once we've merged our branches.

#### What are the relevant tickets?
Closes #47 #39
#### Screenshots (if appropriate)
First two questions:
![Screen Shot 2020-10-14 at 8 57 04 PM](https://user-images.githubusercontent.com/62770843/96071920-4dcf1b80-0e60-11eb-9baf-e965d18ca2bb.png)

#### Questions:
Do you think we have enough questions on the survey?

### New Feature Submissions:
* [x] Have you added an explanation of what your changes do?
### Testing:
